packages/input_schema/src/types.ts

```diff
@@ -0,0 +1,93 @@
+type CommonFieldDefinition<T> = {
+    title: string;
+    description: string;
+    default?: T;
+    prefill?: T extends boolean ? never : T;
```

Comment:
didn't we add support for boolean prefills recently? found this PR as well as some messages on slack on this topic

https://github.com/apify/apify-core/pull/16390

Reply:
Ok, maybe it should be added to our docs. I will remove it there. Same with docs and editor: 'json' option in string type (it is currently missing there, even though we maybe accept it?). Maybe @TC-MO ?